package.json

```diff
@@ -69,5 +69,6 @@
     "ts-node": "^10.9.1",
     "type-fest": "^3.13.0",
     "typescript": "^5.5.3"
-  }
+  },
+  "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e"
```

Comment:
Just out of curiosity, what does this buy us? I've used pnpm in the past but I admit I never used it like this.

Reply:
removed /w rebase.